### PR TITLE
[MOB-5938] Fix response payload size

### PR DESCRIPTION
**Pull request by ahmed1hisham**
## Summary of Changes
- Fixed response payload size value

## Checklist
- [ ] In addition to adding the new API to Dart, I have also added its implementation to both iOS and Android.
- [ ] I have added at least 1 unit test in Dart for the new API and made sure the tests pass locally.
- [ ] I have updated [README.md](https://github.com/Instabug/Instabug-Flutter/blob/master/README.md) to add the new API.

### CODE REVIEW

**Review by AliAbdelfattah — APPROVED** [review]

File: android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java [hunk 1]
```diff
@@ -1122,6 +1122,9 @@ public void apmNetworkLogByReflection(HashMap<String, Object> jsonObject) throws
         //--------------------------------------------
         final String requestContentType = (String) jsonObject.get("contentType");
         final String responseContentType = (String) jsonObject.get("responseContentType");
+        //--------------------------------------------
+         final long requestBodySize = ((Number) jsonObject.get("requestBodySize")).longValue();
+         final long responseBodySize = ((Number) jsonObject.get("responseBodySize")).longValue();
```
Comment (AliAbdelfattah) [inline android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java:1127]: @ahmed1hisham many parsing and casting operations are being done here. Could we please wrap the whole function in a try-catch block?

**Review by AliAbdelfattah — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java (hunk 1 above)
Comment (ahmed1hisham) [reply android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java:1127]: Sure, on it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## v10.8.1 (2021-09-14)
```
[the remark this answers is not in the document]
Comment (ahmed1hisham) [reply CHANGELOG.md]: Done

**Review by AliAbdelfattah — APPROVED** [review]

**Review by AliAbdelfattah — COMMENTED** [review]
@ahmed1hisham please add more test coverage 🙏

**Pull request merged**